crates/metadata-server/src/raft/network/connection_manager.rs

```diff
@@ -122,6 +127,11 @@ where
                         Some(message) => {
                             match message {
                                 Ok(mut message) => {
+                                    counter!(METADATA_SERVER_EMBEDDED_RECV_MESSAGE_TOTAL, "peer" => self.remote_peer.to_string())
+                                        .increment(1);
+                                    counter!(METADATA_SERVER_EMBEDDED_RECV_MESSAGE_BYTES, "peer" => self.remote_peer.to_string())
+                                        .increment(message.payload.len() as u64);
```

Comment:
Let's remove the peer address, this is literally infinite cardinality as those sockets get random ports

Reply:
This only holds the plain node id. But if it doesn't add any value i can remove it.

Reply:
If it's not the address, then it's okay.